### PR TITLE
Finish messages styles

### PR DESCRIPTION
**Pull request by DimaSmagulov**
Styles for flash messages

### CODE REVIEW

File: app/assets/stylesheets/custom.css.scss [hunk 1]
```diff
@@ -164,3 +164,32 @@ input, textarea, select, .uneditable-input {
...
+
+/*flash messages*/
+.alert-error {
+    background-color: #f2dede;
+    border-color: #eed3d7;
+    color: #b94a48;
+    text-align: left;
+ }
+
+.alert-alert {
```
Comment (ruba-ruba) [inline app/assets/stylesheets/custom.css.scss]: `.alert-alert` and `.alert-error` is similar
you can define it once, see => http://stackoverflow.com/a/2099263/2819006

File: app/assets/stylesheets/custom.css.scss [hunk 2]
```diff
@@ -164,3 +164,32 @@ input, textarea, select, .uneditable-input {
...
+    background-color: #f2dede;
+    border-color: #eed3d7;
+    color: #b94a48;
+    text-align: left;
+ }
+
+.alert-success {
```
Comment (ruba-ruba) [inline app/assets/stylesheets/custom.css.scss]: same comment as above

File: app/assets/stylesheets/custom.css.scss (hunk 1 above)
Comment (DimaSmagulov) [inline app/assets/stylesheets/custom.css.scss]: Fixed and squashed

File: app/assets/stylesheets/custom.css.scss [hunk 3]
```diff
@@ -191,3 +191,18 @@ footer {
 p.copyright {
   margin: 15px 0 15px;
 }
+
+/*flash messages*/
+.alert-error, .alert-alert {
+    background-color: #f2dede;
```
Comment (serhiym) [inline app/assets/stylesheets/custom.css.scss]: use 2 spaces

**Pull request merged**